examples/dws/batch_create_cluster_tags.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+"""
+Batch create tags for a specified DWS cluster.
+"""
+import openstack
+
+openstack.enable_logging(True)
+conn = openstack.connect(cloud='otc')
+

```

Comment:
add please here:  sdk.register_otc_extensions(conn)
also you will need to add this import: from otcextensions import sdk

this comment is for all examples

Reply:
Add the following to all examples:

<>
from otcextensions import sdk
<>
sdk.register_otc_extensions(conn)
<>